### PR TITLE
Remove Itanium (IA64) support

### PR DESCRIPTION
**Pull request by richsalz**
I also removed some of the VMS IA64 targets, which is redundant since the VMS stuff is being removed piecemeal, but c'est la vie.

## Checklist
Thank you for your contribution. Please answer the following:

- [X] I acknowledge that I am authorized to submit this code under
the terms of the [Apache License](https://www.apache.org/licenses/LICENSE-2.0)
- [ ] All new public APIs are documented
- [ ] All new public functions have tests
- [ ] All new public functions have fuzzing tests

### CODE REVIEW

**Review by wbl — APPROVED** [review]

**Comment by richsalz** [comment]
Had to fix the conflicts after #151 was merged.  All I did was the rebase, needs another approval.

**Review by wbl — APPROVED** [review]

**Pull request merged**
